docs/cvl/types.md

```diff
@@ -39,27 +39,24 @@ cvl_type ::= basic_type
 See {doc}`basics` for the `id` and `number` productions.
 
 
+(solidity-types)=
 Solidity Types
 --------------
 
-CVL currently supports the following [solidity types][]:
+You can declare variables in CVL of any of the following [solidity types][]:
 
- * `bool`, `int`, `uint`, and the sized variants such as `uint256` and `int8`.
- * `address`
- * `string`, `bytes`, and the sized `bytes` variants (`bytes1` through `bytes32`)
- * Tuples
+ * `int`, `uint`, and the sized variants such as `uint256` and `int8`
+ * `bool`, `address`, and the sized `bytes` variants (`bytes1` through `bytes32`)
+ * `string` and `bytes`
  * {ref}`Single-dimensional arrays <arrays>` (both statically- and dynamically-sized)
- * {ref}`Enum types, struct types, and type aliases <user-types>`
+ * {ref}`Enum types, struct types, and type aliases <user-types>` that are
+   defined in Solidity contracts.
 
-The following are unsupported:
+The following are not directly supported in CVL, although you can interact with
+contracts that use them (see {ref}`type-conversions`):
  * Function types
- * Structs with array members
- * References
  * Multi-dimensional arrays
```

Comment:
@jtoman's work will support this and is already in beta. There are some limitations though.

Reply:
Do you mean multi-dim arrays or the removed lines?

Reply:
multi-dim